src/main/webapp/js/feed-loader.js

```diff
@@ -0,0 +1,54 @@
+
+// Fetch messages and add them to the page.
+function fetchMessages() {
+    const url = '/feed';
+    fetch(url).then((response) => {
+        console.log(response);
```

Comment:
Do you want to keep this `console.log` or what it just for debugging purposes?

Reply:
This is for debugging, I'll remove it. Plus, where should I post the screenshot?